src/main/java/io/jenkins/plugins/ml/Server.java

```diff
@@ -132,6 +135,34 @@ public FormValidation doCheckMaxResults(@QueryParameter String maxResults) {
             }
             return FormValidation.ok();
         }
+
+        /**
+         * Validate the server by testing it
+         */
+        @POST
+        public FormValidation doValidate(
+                                         @QueryParameter String serverAddress,
+                                         @QueryParameter String launchTimeout,
+                                         @QueryParameter String maxResults
+                                         ) throws Exception {
+
+            if(!serverAddress.equals("")) {
+                try{
+                    IPythonUserConfig userConfig = new IPythonUserConfig(serverAddress, Integer.parseInt(launchTimeout), Integer.parseInt(maxResults));
+                    try{
+                        InterpreterManager interpreterManager = IPythonBuilder.getIPythonInterpreterManager(userConfig);
```

Comment:
@shivaylamba Because needs to access the hashtable in the builder class to check/get the interpreter manager for the configuration.

Reply:
Okay Thanks Loghi, understood

Reply:
I think now that the hashmap is not used, that method can be private again.